contracts/src/v0.8/feeds/DualAggregator.sol

```diff
@@ -916,14 +937,8 @@ contract DualAggregator is OCR2Abstract, OwnerIsCreator, AggregatorV2V3Interface
     override
     returns (uint80 roundId_, int256 answer, uint256 startedAt, uint256 updatedAt, uint80 answeredInRound)
   {
-    // TODO: update this function with implementation from the doc
-
-    if (roundId > type(uint32).max) return (0, 0, 0, 0, 0);
+    if (roundId > latestRound()) return (0, 0, 0, 0, 0);
```

Comment:
i think we'd want to keep the `(roundId > type(unit32).max)` case here, and then have a `revert RoundNotFound` if `roundId > latestRound()`

Reply:
What are we expecting as the max latestRound()?
I expected it lower than type(uint32).max, making the first case unreachable, but maybe I'm wrong

Reply:
I think the current implementation works because _latestRound() returns uint32. So the max value that latestRound() can be is type(uint32).max.